sites/all/modules/reol_app_feeds/src/Helper/ParagraphHelper.php

```diff
@@ -341,12 +341,19 @@ private function getCarousel(\ParagraphsItemEntity $paragraph) {
   private function getArticleCarousel(\ParagraphsItemEntity $paragraph) {
     $list = [];
     // Cf. ereol_article_get_articles().
+
     $query = new EntityFieldQuery();
     $count = _reol_app_feeds_variable_get('reol_app_feeds_frontpage', 'max_news_count', 6);
 
+    $bundle = 'article';
+    // Hack for eReolen Go!
+    if (module_exists('breol_news')) {
+      $bundle = 'breol_news';
+    }
```

Comment:
Did it not see this comment and statement in at PR last week as well? Might need to be moved to a helper function?

Reply:
You've seen it in https://github.com/eReolen/base/pull/201/files. I'm working around the fact that ereolen and ereolengo look very similar and yet are not very similar at all. Adding a helper function, `this_is_ereolengo`, say, will not be that useful, and keeping the hack – and the comment – in the code makes it clear that it is actually a hack.